.release-please-manifest.json

```diff
@@ -0,0 +1 @@
+{".":"1.0.0"}
```

Comment:
Please change this to the current version.

Reply:
👍🏻